README.md

```diff
@@ -10,6 +10,9 @@ With BEM nesting:
 
 ![ecss_bem](https://cloud.githubusercontent.com/assets/654597/5896785/60708c5c-a54c-11e4-963f-9e00ede168c3.gif)
 
+With BEM nesting and class names as comments:
+![ecss_bem_comments](https://raw.githubusercontent.com/jaakritso/ecsstractor/master/eCSStractor-comments.mov.gif)
```

Comment:
Replace image code with `![ecss_bem_comments](eCSStractor-comments.mov.gif)`. This way it will link to the image in the correct repository.